packages/foreman/rubygem-deep_cloneable/rubygem-deep_cloneable.spec

```diff
@@ -1,77 +1,78 @@
-%{?scl:%scl_package rubygem-%{gem_name}}
-%{!?scl:%global pkgname %{name}}
-
-%define gem_name deep_cloneable
+# template: default
+%global gem_name deep_cloneable
 
+Name: rubygem-%{gem_name}
+Version: 3.2.0
+Release: 1%{?dist}
 Summary: This gem gives every ActiveRecord::Base object the possibility to do a deep clone
-Name: %{?scl_prefix}rubygem-%{gem_name}
-Version: 3.0.0
-Release: 4%{?dist}
 Group: Development/Languages
```

Comment:
We could drop the group from the template. Fedora has done that already and it doesn't add anything.